connaisseur/flask_server.py

```diff
@@ -26,24 +28,29 @@ def mutate():
         response = admit(admission_request)
     except BaseConnaisseurException as err:
         logging.error(str(err))
-        msg = err.message
+        msg = err.get_user_msg()
```

Comment:
if `BaseConnaisseurException.get_user_msg()` is turned into a property as proposed above, this becomes `msg = err.user_msg` (no parentheses). Also, this assignment is unnecessary; you can fetch the message a few lines down where you actually need it.